### DISCUSSION TITLE
Make keywords visible in screening process

### DISCUSSION THREAD

**Discussion by DesireeFSc** [asreview/asreview#1244] [Q&A]
Hello,
I hope this is not mentioned somewhere already: For my systematic review, it would be beneficial to not only see the abstract during the screening but also to see the keywords the authors used. The keywords are in the ris. file, however, they are not shown while screening. Would it be possible to display them as well?
Thanks in advance!

**Comment by J535D165** [comment]
Currently, this is not supported out of the box. 

Alternatively, you can create a new dataset where you overwrite the abstract with the keywords **and** the abstract. In this case, the model trains on the keywords as well. I can provide a simple code snippet if this solution is useful.

**Reply by DesireeFSc** [reply]
Yes, that would be great! Thank you for your help!

**Reply by J535D165** [reply]
Do you use RIS import files or tabular data like CSV or XLSX?

**Reply by DesireeFSc** [reply]
RIS

**Reply by J535D165** [reply]
https://gist.github.com/J535D165/6ac038da12c041c018094ebb80ac4597 This is an example, let me know if you need any help.
